code/__HELPERS/_macros.dm

```diff
@@ -39,7 +39,7 @@
 
 #define iscluwne(A) istype(A, /mob/living/simple_animal/hostile/retaliate/cluwne)
 
-#define isAI(A) istype(A, /mob/living/silicon/ai)
+#define isAI(A) (istype(A, /mob/living/silicon/ai) || (isobserver(A) && ticker.current_state == GAME_STATE_FINISHED)) 
```

Comment:
@clusterfack not really, examples are things like APC and air alarms, ghosts still can't fuck with them because it doesn't unlock because they're not AIs.